docs/README.md

```diff
@@ -55,12 +55,20 @@ All GraphQL queries and mutations used by Checkout IO are located in [`checkout-
 - [checkout](https://github.com/vtex-apps/checkout)
 - [checkout-container](https://github.com/vtex/checkout-container)
 - [checkout-cart](https://github.com/vtex-apps/checkout-cart)
-- [checkout-identification](https://github.com/vtex-apps/checkout-identification)
+- [product-list](https://github.com/vtex-apps/product-list)
+- [shipping-calculator](https://github.com/vtex-apps/shipping-calculator)
 - [checkout-summary](https://github.com/vtex-apps/checkout-summary)
 - [checkout-coupon](https://github.com/vtex-apps/checkout-coupon)
 - [checkout-step-group](https://github.com/vtex-apps/checkout-step-group)
-- [product-list](https://github.com/vtex-apps/product-list)
-- [shipping-calculator](https://github.com/vtex-apps/shipping-calculator)
+- [checkout-identification](https://github.com/vtex-apps/checkout-identification)
+- [checkout-profile](https://github.com/vtex-apps/checkout-profile)
+- [checkout-shipping](https://github.com/vtex-apps/checkout-shipping)
+- [checkout-payment](https://github.com/vtex-apps/checkout-payment)
 
 ## BFF
 - [checkout-graphql](https://github.com/vtex/checkout-graphql)
+- [places-graphql](https://github.com/vtex-apps/places-graphql)
+- [gateway-graphql](https://github.com/vtex/gateway-graphql)
```

Comment:
sorted alphabetically? 😬 

```suggestion
- [gateway-graphql](https://github.com/vtex/gateway-graphql)
- [places-graphql](https://github.com/vtex-apps/places-graphql)
```

Reply:
The lists seem to be kind of sorted by "importance", like "main" components are being displayed above child components. I'm not sure what is best but I'm not against the way it is right now.

I'd love if the blank line on line 42 was removed, though.